Gruntfile.js

```diff
@@ -196,36 +203,39 @@ module.exports = function( grunt ) {
 
 	grunt.registerTask( "lint", [
 		"jsonlint",
-		runIfNewNode( "eslint:dev" ),
-		runIfNewNode( "eslint:dist" )
+		runIfNewNode( "eslint" )
 	] );
 
-	grunt.registerTask( "test_fast", [ runIfNewNode( "node_smoke_tests" ) ] );
+	grunt.registerTask( "newer:lint", [
+		"newer:jsonlint",
+		runIfNewNode( "newer:eslint" )
+	] );
 
-	grunt.registerTask( "test", [ "test_fast" ].concat(
-		[ runIfNewNode( "promises_aplus_tests" ) ]
-	) );
+	grunt.registerTask( "test:fast", runIfNewNode( "node_smoke_tests" ) );
+	grunt.registerTask( "test:slow", runIfNewNode( "promises_aplus_tests" ) );
+
+	grunt.registerTask( "test", [
+		"test:fast",
+		"test:slow"
+	] );
 
-	// Short list as a high frequency watch task
 	grunt.registerTask( "dev", [
-			"build:*:*",
-			runIfNewNode( "newer:eslint:dev" ),
-			"uglify",
-			"remove_map_comment",
-			"dist:*"
-		]
-	);
+		"build:*:*",
+		"newer:lint:dev",
+		"newer:uglify",
+		"remove_map_comment",
+		"dist:*",
+		"newer:compare_size"
```

Comment:
I'm not sure this is necessary. I want to see the size results of all files every time.